content/news/release-20151224.md

```diff
@@ -0,0 +1,61 @@
+<script>
+ (function(d, t) {
+    var g = d.createElement(t),
+        s = d.getElementsByTagName(t)[0];
+    g.src = 'http://assets.gfycat.com/js/gfyajax-0.517d.js';
+    s.parentNode.insertBefore(g, s);
+}(document, 'script'));
+</script>
```

Comment:
I could, until I added the style attribute to fix the size on the <img> tag. I assume that gfycat has a bug in their script. The .gif is 600x300, as usual, but gfycat makes a <div> with a size of 600x3000.